.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules
 yarn-error\.log
 
 packages/server/coverage/
+
+packages/server/junit\.xml
```

Comment:
```suggestion
```

You have already igored `junit.xml` in the `package/server/.gitingore`. Don't repeat yourself!